commit 029e3ce5ad2220301fe84dc4ef37ccaca5249730
Author: Nirbheek Chauhan <nirbheek@centricular.com>
Date:   Fri Dec 30 16:46:12 2022 +0530

    meson: Don't use outdated 'fallback:' kwarg to dependency()
    
    The mapping is done using the wrap file now. Also update the libffi
    and pcre2 subprojects at the same time.
    
    Reported by Benjamin Gilbert at
    https://gitlab.gnome.org/GNOME/glib/-/merge_requests/3172

diff --git a/meson.build b/meson.build
index 2cdc184ad8c4..f6a07643b7ff 100644
--- a/meson.build
+++ b/meson.build
@@ -2055,7 +2055,7 @@ endif
 
 # Try again with the fallback
 if not pcre2.found()
-  pcre2 = dependency('libpcre2-8', version: pcre2_req, allow_fallback: true, fallback: ['pcre2', 'libpcre2_8'])
+  pcre2 = dependency('libpcre2-8', version: pcre2_req, allow_fallback: true)
   use_pcre2_static_flag = true
 elif host_system == 'windows'
   pcre2_static = cc.links('''#define PCRE2_STATIC
@@ -2078,7 +2078,7 @@ subproject('gvdb')
 gvdb_dep = dependency('gvdb')
 
 libm = cc.find_library('m', required : false)
-libffi_dep = dependency('libffi', version : '>= 3.0.0', fallback : ['libffi', 'ffi_dep'])
+libffi_dep = dependency('libffi', version : '>= 3.0.0')
 
 libz_dep = dependency('zlib')
 
